src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -176,7 +186,23 @@ def get_tags(
     if tags and isinstance(tags, list):
         tags = set(tags[0])
 
-    tags = tags | {"level", "release", "handled", "environment"}
+    default_tags = {"level", "release", "handled", "environment"}
+    # for performance issues we want to have the default tags _except_ level
+    if (
+        group.issue_category == GroupCategory.PERFORMANCE
+        and group.issue_type not in REGRESSION_PERFORMANCE_ISSUE_TYPES
+    ):
+        default_tags.remove("level")
```

Comment:
Regression performance issues should not have any tags at all, Does this if case handle that?

Reply:
cc @ceorourke

Reply:
This is removing the `level` tag for all performance issues except for the regression performance issues, and then the code block below it removes _all_ tags for regression performance issues and issue categories that are not error or performance.